### PR TITLE
Added playbook for new role debops.apt_cacher_ng.

### PR DESCRIPTION
**Pull request by ypid**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -35,6 +35,11 @@ v0.2.9
 
 - Moved roles dependencies from ``debops.subnetwork`` to subnetwork playbook. [ypid]
 
+- Add ``debops.apt_cacher_ng`` role to it's own
+  ``service/unattended_upgrades.yml`` playbook. The server-side support for the
```
Comment (drybjed) [inline CHANGELOG.rst]: Looks like wrong playbook name. :-)

**Comment by drybjed** [comment]
You would also need to add the role to the Galaxy requirements. But first, let me move it to the `debops/` organization. Role looks very nice, by the way.

**Comment by ypid** [comment]
All right. Thanks. I can update this PR later.

**Comment by drybjed** [comment]
Role is in Galaxy: https://galaxy.ansible.com/debops/apt_cacher_ng/

**Comment by ypid** [comment]
Thanks. Updated PR.

**Comment by drybjed** [comment]
Hmm, merge conflicts? Weird...

**Comment by ypid** [comment]
Ah, I did not properly rebase … fixed.

**Comment by ypid** [comment]
Passed :wink:

**Pull request merged**